mmdet/models/detectors/single_stage_instance_seg.py

```diff
@@ -95,7 +95,12 @@ def _forward(self,
             mask_outs = self.mask_head.forward(x)
         else:
             mask_outs = self.mask_head.forward(x, positive_infos)
-        outs = outs + (mask_outs, )
+        # YOLACT segmentation branch,
```

Comment:
If the child's logic is different from the parent, you should overwrite this method in the child instead of modify the parent.